source/includes/fact-non-utf8-data.rst

```diff
@@ -0,0 +1,17 @@
+If Compass encounters data display issues when querying your data 
+with the error message: ``Invalid UTF-8 string in BSON document``. You 
```

Comment:
(question)
I think the error message you showed me was a bit longer, no? Consider adding the whole thing, for those folks who might copy the error message and google it. 

(question)
What happens if you ignore this message? Why are you getting the error? What happens as a result of this error? Etc.

Reply:
This is the error message linked in the support ticket, there is a slightly larger message `An error occured while loading instance info: Invalid.... `, I can confirm which one to use in the TR. The content has been updated to address your other q's.